File: tests/fixtures/code_sample/goapp/billing/billing.go
// Package billing totals orders and sends their invoices.
package billing

import "fmt"

// Total adds up an order's lines.
func Total(order Order) int {
	amount := 0
	for _, line := range order.Lines {
		amount += line
	}
	return amount
}

// SendInvoice mails the invoice for one order.
func SendInvoice(order Order) {
	fmt.Println("sending", order.ID)
}

File: tests/fixtures/code_sample/csapp/Billing/Billing.cs
namespace CsApp.Billing;

/// <summary>Billing totals orders and sends their invoices.</summary>
public static class Billing
{
    /// <summary>Total adds up an order's lines.</summary>
    public static int Total(Order order)
    {
        var amount = 0;
        foreach (var line in order.Lines)
        {
            amount += line;
        }

        return amount;
    }

    /// <summary>SendInvoice mails the invoice for one order.</summary>
    public static void SendInvoice(Order order)
    {
        Console.WriteLine("sending", order.Id);
    }
}
